apps/filecoin-site/src/app/_styles/globals.css

```diff
@@ -493,7 +490,7 @@
 
   /* TOOLTIP */
   .tooltip {
-    @apply rounded-xl border border-[var(--color-border)] bg-white shadow-sm;
+    @apply rounded-xl border border-zinc-200 bg-white shadow-sm;
```

Comment:
Filipa mentioned the following:

<img width="556" height="530" alt="Image" src="https://github.com/user-attachments/assets/8c5b2a75-8a70-40ac-b242-869124a1c218" />